libgoal/libgoal.go

```diff
@@ -952,6 +978,8 @@ func (c *Client) AddParticipationKey(keyfile string) (resp generated.PostPartici
 	}
 
 	return algod.PostParticipationKey(data)
+
+	// PKI TODO: Install state proof keys here.
```

Comment:
Do we really need the TODO here?

If I understand this correctly, the entire old participation db file is being transmitted over the REST API, so state proof keys would get registered.
I think we will need this todo once we use the pure register API (without the transmitting the file)